web/src/engine/osk/src/input/gestures/browser/subkeyPopup.ts

```diff
@@ -317,11 +328,40 @@ export default class SubkeyPopup implements GestureHandler {
       // We're going to adjust the top of the box to ensure it stays
       // pixel aligned, otherwise we can get antialiasing artifacts
       // that look ugly
-      let top = Math.floor(keyRect.top - _BoxRect.top - 9 + delta);
+      let top = Math.floor(
+        Number.parseInt(cms.top, 10) +
```

Comment:
This feels like it should be `strToInt()` or something given how frequently we do it! (Praps a job for another day)